includes/rest-api/class-sensei-rest-api-setup-wizard-controller.php

```diff
@@ -149,7 +160,32 @@ public function register_submit_features_route() {
 							'type'     => 'array',
 							'items'    => [
 								'type' => 'string',
-								'enum' => $this->setup_wizard->plugin_slugs,
+							],
+						],
+					],
+				],
+			]
+		);
+	}
+
+	/**
+	 * Register /features endpoint.
+	 */
+	public function register_submit_features_installation_route() {
```

Comment:
The comments are swapped for features and features-installation in both methods.

Reply:
Oops! https://github.com/Automattic/sensei/pull/3190/commits/17626e750d8351c8677134a9eaf7ba5086738025